tests/platform_helper/test_command_codebase.py

```diff
@@ -172,13 +237,13 @@ def test_codebase_deploy_does_not_trigger_build_without_an_application(
                 "dev",
                 "--codebase",
                 "application",
-                "--commit",
+                "--ref",
                 "ab1c23d",
```

Comment:
Probably the case in other tests as well, but the tests also act as documentation of sorts, so would be better with more real=life examples of what we're expecting. i.e. here:
```suggestion
                "commit-ab1c23d",
```

Reply:
Done.